Assignment1/Resume.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html>
+    <head>
+<title>Resume</title>
+<link rel="stylesheet" href="style.css" type="text/css"/>
+<link rel="stylesheet" href="StyleSheet.css" type="text/css"/>
+</head>
+    <body>
+        <div class="container">
+<div class="container1">
+<div class="namingData"><center><table class="tableHeader">
+    <tr class="row">
+        <td class="data"><span class="name">TEAM ORANGE</span><br/><span class="designation"><i>Software Developer Trainee</i></span> </td>
```

Comment:
keep it in separate lines so it will be readable to other developers. 

Why there is a space before </td> tag